static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-love-boat.js

```diff
@@ -0,0 +1,143 @@
+define([
+    'bean',
+    'common/modules/commercial/contributions-utilities',
+    'common/utils/template',
+    'common/utils/config',
+    'common/utils/$',
+    'common/views/svgs',
```

Comment:
imo the most user friendly thing would be to export an object of the compiled svgs, so i can just import `svgs` and later use `svgs.hand`